vue-components/package.json

```diff
@@ -14,7 +14,7 @@
     "build:main": "vue-cli-service build --target lib --formats commonjs --name wikit-vue-components src/main.ts --report --report-json",
     "build:tsdecl": "tsc --emitDeclarationOnly",
     "build:vue3compat": "VUE_APP_VUE3COMPAT=true vue-cli-service build --target lib --formats commonjs --name wikit-vue-components-vue3compat --no-clean src/main.ts",
-    "test:unit": "vue-cli-service test:unit",
+    "test:unit": "vue-cli-service test:unit -u",
```

Comment:
`-u` should not be part of the regular test command.

Reply:
Can you please offer more constructive review? What can be another way of updating the snapshot?

Reply:
`-u` is the right way to update the snapshot, but it shouldn’t have been committed. Either remove it from the `package.json` again before creating the commit, or run something like `npm run test:unit -u` or `npx jest  -u` manually.

Though when I try it now, Jest seems to update the snapshot even without the `-u` option, which I find quite confusing – no idea what’s going on there…